certbot-apache/certbot_apache/tests/testdata/debian_apache_2_4/multiple_vhosts/apache2/sites-available/ocsp-ssl.conf

```diff
@@ -0,0 +1,36 @@
+<IfModule mod_ssl.c>
```

Comment:
You have two copies of this file - the one under letsencrypt-apache probably needs to go.